Automattic/jetpack
# Admin Bar: Link Stats to Calypso

Fixes Automattic/wp-calypso#86959

## Proposed changes:
* Links the Stats item to Calypso, where there are more advanced stats. 

## Jetpack product discussion
See original issue. I have no strong views on this - simply addressing the open High Priority issue. 

## Does this pull request change what data or activity we track or use?
No. 

## Testing instructions:
In order for the Admin Bar to appear, you'll need to be signed in. Then, you'll want to click this icon on the Admin Bar and verify that you're directed to the Calypso interface.

<img width="940" alt="Screenshot 2024-02-04 at 19 55 25" src="https://github.com/Automattic/jetpack/assets/43215253/0f705793-f95a-4f5a-8b91-d1cc3399c69b">
